examples/model_compress/slim_pruner_torch_vgg19.py

```diff
@@ -0,0 +1,176 @@
+import math
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchvision import datasets, transforms
+from nni.compression.torch import SlimPruner
+
+
+class vgg(nn.Module):
```

Comment:
1. Vgg19 is similiar as vgg16,  is it possible to just demostrate one of the two models?
2.  Can we leverage torch.hub or torchvision.models  to emphysize the pruner instead of model structure?

Reply:
1. Yes they are similar, but we want to keep the experiments consistent with the original papers , as they use vgg16 and vgg19 separately.
2. torchvision.models are for ImageNet dataset, which is too large for users to test the algorithm, I'll investigate torch.hub later.
Thanks for your review